File: arcgis_map_sdk_ios/ios/Classes/Models/UserPosition.swift
//
//  UserPosition.swift
//  arcgis_map_sdk_ios
//
//  Created by Julian Bissekkou on 27.11.23.
//

import Foundation

struct UserPosition: Codable {
    let latLng: LatLng
    let accuracy: Double?
    let heading: Double?
}

File: arcgis_map_sdk_platform_interface/lib/src/models/user_position.dart
import 'package:arcgis_map_sdk_platform_interface/arcgis_map_sdk_platform_interface.dart';

class UserPosition {
  final LatLng latLng;
  final double? accuracy;
  final double? velocity;
  final double? heading;

  const UserPosition({
    required this.latLng,
    this.accuracy,
    this.velocity,
    this.heading,
  });
}
